connectors/socialcast/src/main/java/com/vmware/connectors/socialcast/SocialcastRequestContext.java

```diff
@@ -89,7 +89,7 @@ public String getResultJson() throws JsonProcessingException {
         return JSON_WRITER.writeValueAsString(resultsData);
     }
 
-    @SuppressWarnings("MismatchedQueryAndUpdateOfCollection")
+    @SuppressWarnings({"MismatchedQueryAndUpdateOfCollection", "PMD.UseConcurrentHashMap"})
```

Comment:
Controversial again.

Reply:
I am ignoring this rule in pmdrules.xml. I have restored it to ```HashMap```.